### PR TITLE
UN updates

### PR DESCRIPTION
**Pull request by mcvholloway**
Summary of major changes:
- Condensed some of the directions into fewer steps. Eg. Instead of having checking the top and bottom five rows as separate steps after merging, this is just part of the directions for that question. Also, cut down on some of the repetitive questions towards the end.
- Added a link to the read_csv documentation when they get to the step to read in the internet users data. 
- Removed the directions to subset to just three years and did not direct to create three new subset dataframes, except at the very end in order to look at just 2014 data. 
- Added some analysis of the datasets on their own before merging them and some quick questions to answer to give more practice on basic EDA questions. 
- Included questions where value_counts can be used (number of rows per year and per country). 
- Includes a link to Python Graph Gallery on the first question where they are asked to create a plot.
- Added a question at the end where they need to assign a group using qcut. The purpose of this question is to be able to have them use a .groupby to compare median internet users by group.

### CODE REVIEW

**Review by MWeissSharp — COMMENTED** [review]
I think this looks great! What do you guys think about actually guiding them to do different plot types for #8 (histogram, density plot, box plot) and  #10 (bar, line, scatter) and ask them which plot makes the most sense for the data?

File: python_for_analytics/projects/GDP-and-Internet-Usage/README.md [hunk 1]
```diff
@@ -13,51 +10,37 @@
...
-96. Repeat this one more time to look at 10-year trend for the 5 countries for 2014 with the lowest GDP per capita.
-30.	Is there anything surprising or unusual in any of these plots? Searching on the internet, can you find any possible explanations for unusual findings?
-
+4.	Using the pandas `read_csv()` method, read the GDP dataset into your notebook as a DataFrame called `gdp_df`. Take a look at the first few and last few rows to familiarize yourself with what is contained in this dataset.
+5. How many rows and columns are in `gdp_df`? What are the data types of each column?
+6. Drop the `Value Footnotes` column and rename the remaining three to 'Country', 'Year', and 'GDP_Per_Capita'.
+7. How many countries have data for all years? Which countries are missing many years of data? Look at the number of observations per year. What do you notice? 
+8. Create a plot to show the distribution of GDP per capita for the year 2020. If you're unsure of what type of plot to use, explore the Python Graph Gallery (https://www.python-graph-gallery.com/). What do you notice when you look at this plot?
```
Comment (mcvholloway) [inline python_for_analytics/projects/GDP-and-Internet-Usage/README.md]: Something like this?

```suggestion
8. In this question, you're going to create some plots to show the distribution of GDP per capita for the year 2020. Go to the Python Graph Gallery (https://www.python-graph-gallery.com/) and look at the different types of plots under the Distribution section. Create a histogram, a density plot, a boxplot, and a violin plot. What do you notice when you look at these plots? How do the plots compare and what information can you get out of one type that you can't necessarily get out of the others?
```

File: python_for_analytics/projects/GDP-and-Internet-Usage/README.md [hunk 2]
```diff
@@ -13,51 +10,37 @@
...
+9. What was the median GDP per capita value in 2020?
+10. Create a boxplot to compare the distribution of GDP per capita values for 1990, 2000, 2010, and 2020. Your plot should have Year on the x-axis and GDP_Per_Capita on the y-axis. Comment on what you observe has happened to GDP values over time.
```
Comment (mcvholloway) [inline python_for_analytics/projects/GDP-and-Internet-Usage/README.md]: And this?
```suggestion
10. For this question, you're going to create some visualizations to compare GDP per capita values for the years 1990, 2000, 2010, and 2020. Start by subsetting your data to just these 4 years into a new DataFrame named gdp_decades. Using this, create the following 4 plots:
	* A boxplot
	* A barplot (check out the Barplot with Seaborn section: https://www.python-graph-gallery.com/barplot/#Seaborn)
	* A scatterplot
	* A scatterplot with a trend line overlaid (see this regplot example: https://www.python-graph-gallery.com/42-custom-linear-regression-fit-seaborn)
Comment on what you observe has happened to GDP values over time and the relative strengths and weaknesses of each type of plot.
```

File: python_for_analytics/projects/GDP-and-Internet-Usage/README.md (hunk 1 above)
Comment (MWeissSharp) [reply python_for_analytics/projects/GDP-and-Internet-Usage/README.md]: Awesome!

File: python_for_analytics/projects/GDP-and-Internet-Usage/README.md (hunk 2 above)
Comment (MWeissSharp) [reply python_for_analytics/projects/GDP-and-Internet-Usage/README.md]: Yes, exactly!

**Review by MWeissSharp — APPROVED** [review]
Very excited about all of this!

**Pull request merged**